README.md

```diff
@@ -1,7 +1,10 @@
 # SystemLink Operations Handbook
 
 This repository contains documentation, examples, and example config files for
-IT professionals to use when managing an NI SystemLink Server installation.
+IT professionals to use when managing an NI SystemLink Server installation
+beyond the basic configuration installed with SystemLink. For information about
```

Comment:
>when managing an NI SystemLink Server installation beyond the basic configuration installed with SystemLink.

Would something like "when managing more advanced configurations of NI SystemLink Server" work? Idk if that's technically as accurate as it could be...but like, instead of saying what this handbook goes beyond, saying what the handbook actually addresses. This would apply in the second instance below too.

Reply:
Yeah, I like that